src/views/DEPRECIATED-workordersnew.php

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
You can just delete this file. That will remove it from this revision. We can always go back to an earlier commit if we need the file.